tasks/Estimating Geological Metrics.md

```diff
@@ -0,0 +1,69 @@
+# Summary
+
+Participants were organized into groups of four and performed a series of tasks in a series of rounds with old and new group members. This task was an intellective task. participants were asked to correctly answer a series of questions without looking up the solutions online, instead debating with their group members on what the answers to the questions could be. At the end teams decided on the correct answer for each prompt and one person submitted a list on behalf of the team 
+
```

Comment:
"Participants were asked to correctly answer a series of questions without
looking up the solutions online, instead debating with their group members on what the answers
to the questions could be. Prompts included estimating the number of states that border the
Gulf of Mexico, the percentage of the U.S. population that goes online at least once a week, and
the height in feet of Mount McKinley. At the end of each round, the team collectively decided on
their final answers and one member submitted a final list of answers on behalf of the team."